exporter/kafkaexporter/go.mod

```diff
@@ -22,6 +22,7 @@ require (
 
 require (
 	github.com/apache/thrift v0.17.0 // indirect
+	github.com/benbjohnson/clock v1.3.0 // indirect
```

Comment:
Did this come up in a `go mod tidy` ?

Reply:
I swear `make gotidy` added it, but now it doesn't.  I'll make sure it's pulled in the next iteration.

Reply:
Not to worry, `go mod tidy` does so magic at times.